modules/config/pages/deploy.adoc

```diff
@@ -344,7 +344,7 @@ Password: XXXXX
 [subs="+quotes,+macros"]
 ----
 [standalone@localhost:19990 /] deploy \
-/home/vagrant/eap-qc-apps/admin1/temperature-converter.war
+/home/vagrant/eap-qc-apps/admin1/material/temperature-converter.war
```

Comment:
do not check WAR files into git directly. We should host WAR files in github releases. i'll show you tomorrow

Reply:
please don't create a fork when you can create local branches. The HTML gen task is disabled for clones.